rbac/migration_tool/migrate.py

```diff
@@ -155,6 +155,10 @@ def migrate_data_for_tenant(tenant: Tenant, exclude_apps: list, replicator: Rela
 
 def migrate_data(exclude_apps: list = [], orgs: list = [], write_relationships: str = "False"):
     """Migrate all data for all tenants."""
+    if not settings.READ_ONLY_API_MODE:
+        logger.info("Read-only API mode is required. READ_ONLY_API_MODE must be set to true.")
```

Comment:
```suggestion
        logger.fatal("Read-only API mode is required. READ_ONLY_API_MODE must be set to true.")
```

Reply:
I'd argue it's fatal because it does not function at all – if you expected it to migrate, it didn't.